.github/workflows/node-ci.yml

```diff
@@ -35,10 +35,13 @@ jobs:
             ${{ runner.OS }}-
       - name: Install dependencies
         run: yarn install --frozen-lockfile
-      - name: Run tests
+      - name: Check code quality
+        if: ${{ matrix.node-version == '15.x' }}
```

Comment:
Please make it a separate Job (it will then run in parallel to the tests).
You can basically copy the `canarist` block below, I think.

Reply:
Thought about that. But then I think I have to copy all the installation & caching ceremony for roughly 12 seconds, that this one job would take longer otherwise.